data/supporters.json

```diff
@@ -430,16 +430,16 @@
       ]
     },
     {
-      "name": "Wizard Amigos Institute",
+      "name": "WizardAmigos",
```

Comment:
Is the missing space on purpose? In line https://github.com/rubyberlin/code-of-conduct/pull/107/files#diff-018defbdea6a414eac9adadf92c35e12R440 there’s is a space.